title:  UCLouvain-CBIO BSS2020
---

# Bioinformatics Summer School 2020

> - [UCLouvain](http://uclouvain.be/), Louvain-la-Neuve, Belgium
> - 6 - 10 July 2020
> - [Louvain House](https://uclouvain.be/fr/louvainhouse)

This one-week intensive summer school in bioinformatics will focus on
**data analysis** and **high throughput biology**, with a special
focus on **R**/[**Bioconductor**](https://lgatto.github.io/about/) and
its application to a wide range of topics across bioinformatics and
computational biology. The course is intended for researchers who are
familiar with omics experimental technologies and their applications
in biology, have had some exposure with R, and who want to learn or
expand their bioinformatics skills.

The course will cover underlying theory and practical hands-on
exercises. At the end of the course, participants should be able to
run analysis workflows on their own omics data, adapt and combine
different tools and make informed choices about bioinformatics data
analysis strategies.

## Topics

- Bioconductor core technologies
- Elements of statistics and machine learning
- Omics data visualisation
- High throughput data processing
- RNA Seq data analysis (bulk and single cell)
- Quantitative proteomics
- Reproducible research and workflow authoring with R markdown

Previous edition: [https://uclouvain-cbio.github.io/BSS2019/](https://uclouvain-cbio.github.io/BSS2019/)

## Workshop faculty

**Instructors**

- [**Martin Morgan**](https://www.roswellpark.org/martin-morgan) (Roswell
  Park Comprehensive Cancer Center, head of the Bioconductor team) -
  Bioconductor core technologies.
- [**Laurent Gatto**](https://lgatto.github.io/about/) (de Duve
  Institute, UCLouvain) - R/Bioconductor, Mass spectrometry and
  proteomics.
- [**Lieven Clement**](https://statomics.github.io/) (Statistical Genomics Group, UGent) -
  Statistical genomics and quantitative proteomics.
- [**Charlotte Soneson**](https://csoneson.github.io/index.html)
  (Friedrich Miescher Institute and SIB Swiss Institute of
  Bioinformatics) - experimental design and differential expression,
  and single cell RNA-seq.
- ...
<!-- - [**Oliver Crook**](https://olivercrook.co.uk/) (University of -->
<!--   Cambridge, UK) - Statistics and machine learn. -->

**Teaching assistants**

- Members of the UCLouvain [Computational Biology research
  group](https://lgatto.github.io/cbio-lab/).
<!-- - Members of the [SMCS](https://sites.uclouvain.be/smcs-gateway/) -->
<!--   (Support en Méthodologie et Calcul Statistique) platform. -->
<!-- - Jérôme Ambroise, CTMA, IREC, UCLouvain. -->

<!-- ## Schedule -->

<!-- <table> -->
<!--   <thead> -->
<!-- 	<tr> -->
<!-- 	  <th>Time </th> -->
<!-- 	  <th>Monday</th> -->
<!-- 	  <th>Tuesday</th> -->
<!-- 	  <th>Wednesday</th> -->
<!-- 	  <th>Thursday</th> -->
<!-- 	  <th>Friday</th> -->
<!-- 	</tr> -->
<!--   </thead> -->
<!--   <tbody> -->
<!-- 	 <tr> -->
<!-- 		<th>8.30-9.00 </th> -->
<!-- 		<td colspan=5>Coffee and breakfast (on site)</td> -->
<!-- 	 </tr> -->
<!-- 	 <tr> -->
<!-- 		<th>9.00-9.30 </th> -->
<!-- 		<td>Registration, help desk and opening remarks ([slides](https://docs.google.com/presentation/d/1RK5czd_e-sEx1MguYa093Y2s7d8EECJyGyn0JPJDAUY/edit?usp=sharing))</td> -->
<!-- 		<td rowspan=2>Bioconductor core technologies (SE, Genomics ranges) ([Rmd](https://raw.githubusercontent.com/Bioconductor/BiocIntro/BSS2019/vignettes/03_CoreApproachesInBioconductor.Rmd), [html](https://uclouvain-cbio.github.io/BSS2019/03_CoreApproachesInBioconductor.html)) (MM)</td> -->
<!-- 		<td rowspan=2>Gene set enrichment analysis (with practical) ([slides pdf](https://uclouvain-cbio.github.io/BSS2019/06_GeneSetEnrichmentSlides.pdf), [html](https://uclouvain-cbio.github.io/BSS2019/06_GeneSetEnrichment.html), [Rmd](https://raw.githubusercontent.com/Bioconductor/BiocIntro/BSS2019/vignettes/06_GeneSetEnrichment.Rmd)) (MM)</td> -->
<!-- 		<td rowspan=2>High throughput sequencing data processing ([slides](https://uclouvain-cbio.github.io/BSS2019/HTSProcessing.pdf)) (CS, JM)</td> -->
<!-- 		<td rowspan=2>Single cell RNA-seq [slides](https://uclouvain-cbio.github.io/BSS2019/SingleCellRNAseq_small_pdf.pdf) (KVB)</td> -->
<!-- 	 </tr> -->
<!-- 	 <tr> -->
<!-- 		<th>9.30-10.30 </th> -->
<!-- 		<td>Talk: Bioinformatics at GSK Vaccines, Christophe Lambert</td> -->
<!-- 	 </tr> -->
<!-- 	 <tr> -->
<!-- 		<th>10.30-11.00</th> -->
<!-- 		<td colspan=5>Coffee break</td> -->
<!-- 	 </tr> -->
<!-- 	 <tr> -->
<!-- 		<th>11.00-12.30</th> -->
<!-- 		<td>Introduction to R and Bioconductor ([Rmd](https://raw.githubusercontent.com/Bioconductor/BiocIntro/BSS2019/vignettes/01_IntroToRAndBioconductor.Rmd), [html](https://uclouvain-cbio.github.io/BSS2019/01_IntroToRAndBioconductor.html)) (MM)</td> -->
<!-- 		<td>Visualisation ([html (part 1)](https://htmlpreview.github.io/?https://github.com/lgatto/visualisation/blob/master/graphs.html) and [slides (part 2](https://htmlpreview.github.io/?https://github.com/lgatto/visualisation/blob/master/highdimvis.html#)) (LG)</td> -->
<!-- 		<td>Elements of statistics and machine learning ([slides](https://github.com/ococrook/2019-BSS/raw/master/Intro2statsml.pdf)) (OC)</td> -->
<!-- 		<td>Practical: RNASeq ([Rmd](https://raw.githubusercontent.com/UCLouvain-CBIO/BSS2019/master/docs/rnaseq_gene_summerschool_belgium_2019.Rmd?token=AAC5ZRSJPZB3QU7I2LOGURS5F3SQK) and [html](https://uclouvain-cbio.github.io/BSS2019/rnaseq_gene_summerschool_belgium_2019.html))  </td> -->
<!-- 		<td>Mass spectrometry-based proteomics ([slides](https://lgatto.github.io/bioc-ms-prot/bss-slides.html), vignette [Rmd](https://raw.githubusercontent.com/lgatto/bioc-ms-prot/master/bss-lab.Rmd) [html](https://lgatto.github.io/bioc-ms-prot/bss-lab.html)) (LG)</td> -->
<!-- 	 </tr> -->
<!-- 	 <tr> -->
<!-- 		<th>12.30-14.00</th> -->
<!-- 		<td colspan=5>Lunch (on site)</td> -->
<!-- 	 </tr> -->
<!-- 	 <tr> -->
<!-- 		<th>14.00-15.30</th> -->
<!-- 		<td>Rmarkdown for reproducible research ([slides](https://lgatto.github.io/rr-rmarkdown-intro-slides/index.html)) (LG)</td> -->
<!-- 		<td>Practical: Bioc core technologies ([Rmd](https://raw.githubusercontent.com/Bioconductor/BiocIntro/BSS2019/vignettes/04_Practical_CoreApproachesInBioconductor.Rmd), [html](https://uclouvain-cbio.github.io/BSS2019/04_Practical_CoreApproachesInBioconductor.html))</td> -->
<!-- 		<td>Practical: stats/ML ([Rmd](https://github.com/ococrook/2019-BSS/blob/master/practical/Intro2statmlPractical.Rmd) and [html](https://htmlpreview.github.io/?https://github.com/ococrook/2019-BSS/blob/master/practical/Intro2statmlPractical.html)) (OC)</td> -->
<!-- 		<td>Bulk RNA-Seq ([slides](https://uclouvain-cbio.github.io/BSS2019/BulkRNAseq.pdf)) (CS, JM)</td> -->
<!-- 		<td>Quantitative proteomics data analysis ([slides and practical](https://uclouvain-cbio.github.io/BSS2019/pda.html)) (LC)</td> -->
<!-- 	 </tr> -->
<!-- 	 <tr> -->
<!-- 		<th>15.30-16.00</th> -->
<!-- 		<td colspan=5>Coffee break</td> -->
<!-- 	 </tr> -->
<!-- 	 <tr> -->
<!-- 		<th>16.00-17.30</th> -->
<!-- 		<td>Practical: R/Bioconductor and Rmarkdown ([Rmd](https://raw.githubusercontent.com/Bioconductor/BiocIntro/BSS2019/vignettes/02_Practical_RBioconductorAndReproductibleResearch.Rmd), [html](https://uclouvain-cbio.github.io/BSS2019/02_Practical_RBioconductorAndReproductibleResearch.html))</td> -->
<!-- 		<td>Working with biological  annotations ([Rmd](https://raw.githubusercontent.com/Bioconductor/BiocIntro/BSS2019/vignettes/05_Annotations.Rmd), [html](https://uclouvain-cbio.github.io/BSS2019/05_Annotations.html)) (MM)</td> -->
<!-- 		<td>Principles of differential expression analysis (experimental design and batch effect) ([slides](https://uclouvain-cbio.github.io/BSS2019/ExperimentalDesign.pdf)) (CS)</td> -->
<!-- 		<td>Practical: RNASeq ([Rmd](https://uclouvain-cbio.github.io/BSS2019/rnaseq_gene_summerschool_belgium_2019.Rmd) and [html](https://uclouvain-cbio.github.io/BSS2019/rnaseq_gene_summerschool_belgium_2019.html))</td> -->
<!-- 		<td>Practical: proteomics</td> -->
<!-- 	 </tr> -->
<!-- 	 <tr> -->
<!-- 		<th>17.30-18.00</th> -->
<!-- 		<td>QA and day 1 wrap-up</td> -->
<!-- 		<td>QA and day 2 wrap-up</td> -->
<!-- 		<td>QA and day 3 wrap-up</td> -->
<!-- 		<td>QA and day 4 wrap-up</td> -->
<!-- 		<td>QA and workshop wrap-up</td> -->
<!-- 	 </tr> -->
<!-- </tbody> -->
<!-- </table> -->

<!-- ## Downloads -->

<!-- - [Airways data](https://uclouvain-cbio.github.io/BSS2019/airway.zip) -->
<!--   (installed with the script above). -->
<!-- - The [extdata](https://uclouvain-cbio.github.io/BSS2019/extdata.zip) -->
<!--   files (installed with the script above). -->
<!-- - The [`hiiragi2013` ExpressionSet -->
<!--   object](https://github.com/lgatto/visualisation/raw/master/data/hiiragi2013.rda). -->
<!-- - DESeq2/dgeR objects: -->
<!--   [airway_dge.rds](https://uclouvain-cbio.github.io/BSS2019/airway_dge.rds) -->
<!--   and -->
<!--   [airway_ds_se.rds](https://uclouvain-cbio.github.io/BSS2019/airway_ds_se.rds) -->
<!-- - The [CPTAC proteomics data](https://raw.githubusercontent.com/lgatto/bioc-ms-prot/master/data/cptac_peptides.txt). -->

## Registration

The summer school will be limited to 50 participants.

<!-- Registrations are now closed. -->

<!-- To register, please fill out the [Google -->
<!-- form](https://goo.gl/forms/AqQrRSZTZKBIaFkR2), providing contact -->
<!-- details, information about your experience using R, Bioconductor, and -->
<!-- omics in general, how this workshop will benefit your research, and a -->
<!-- CV and motivation statement in case you would like to qualify for a -->
<!-- scholarship. These information will be used by the organising -->
<!-- committee to select participants in case of -->
<!-- over-subscription. Registrants will then be notified by the 1 April -->
<!-- 2019 whether they were selected and will be invited to proceed with -->
<!-- payment by 15 April, upon which final registration will be confirmed. -->

<!-- Note that after payment, no refunds will be offered in case of -->
<!-- cancellation. -->

## Fees

- Students and post-doctoral researchers: 25 euros
- Permanent academic staff: 200 euros
- Industry: 1000 euros

## Scholarships

We are happy to offer up to 20 scholarships of up to 1000 euros for
travelling and accommodation to students and early career researchers.

## Course material

The course material (lectures and practicals) will be made available
on the course web page.

**Participants are required** to bring their own laptop with the most
recent release versions of R and Bioconductor installed: R-3.6.2 and
Bioconductor 3.10 installed. Please make sure that your computer’s
hardware is sufficiently powered (>4 GB RAM, > 2 GB free disk space),
that you have a working wireless card, and that you have administrator
rights. Please follow these [software installation
instructions](https://uclouvain-cbio.github.io/BSS2019/installation.html)
**before** the workshop.


**Prerequisites** Participants should be familiar to R and have some
experience of high throughput omics data. They should typicall feel at
ease with the suggested preliminary material by Martin Morgan below.

- [R/Bioconductor for open-source analysis and comprehension of high-throughput genomics data](https://docs.google.com/presentation/d/e/2PACX-1vS25aIVGDGCAJMsG1VNIxKaUbU7tsefGwMzAIxBLtUJG8ZO44m0zjYRvajli18xSbzwCgFrU_Zlb9kq/pub?start=false&loop=false&delayms=3000#slide=id.p) (slides)
- [Introduction to R](https://bioconductor.org/help/course-materials/2017/CSAMA/labs/1-monday/lab-01-intro-to-r-bioc/L1.1-r-intro-morgan.html) (lab)
- [Introduction to Bioconductor](https://bioconductor.org/help/course-materials/2017/CSAMA/labs/1-monday/lab-01-intro-to-r-bioc/L1.2-bioc-intro-morgan.html) (lab)


## Accommodations and travel information

- Participants are invited to arrange their travelling and accommodation directly.
- Reaching [Louvain-la-Neuve](https://uclouvain.be/en/sites/louvain-la-neuve/acces-et-plan.html).

**Suggested accommodations**

- [Martin's
  Louvain-la-Neuve](https://www.martinshotels.com/en/hotel/martins-louvain-la-neuve),
  located next to the workshop venue. Preferential room tariffs will
  be provided to participants.
- [Ibis Styles Louvain la Neuve Hotel
  ](https://www.accorhotels.com/gb/hotel-2200-ibis-styles-louvain-la-neuve-hotel-and-events/index.shtml),
  located 15 minutes walk to venue.

## Social programme

All participants are invited to join for dinner in the Louvain House
mezzanine on Wednesday at 7pm.

## Organising committee

- Laurent Gatto
- Axelle Loriot
- Julie Klein

## Contact

For inquiries related to the registration, scientific program,
organisation, please contact us at contact-cbio@uclouvain.be.

## Code of conduct

This code of conduct is developed based on the Bioc2019 conference
code of conduct.

BSS2020 is dedicated to providing a supportive, collegial, and
harassment-free experience for everyone, regardless of:

- gender, gender identity and expression, sexual orientation,
  disability, physical appearance, body size, race, age or religion.
- intellectual position: approaches to data analysis, software
  preferences, coding style, scientific perspective, etc.

We do not tolerate harassment, intimidation, or bullying of
participants in talks, workshops, poster sessions, social activities,
or online. Sexual language and imagery is not
appropriate. Participants violating these rules may be sanctioned or
expelled from the event with no refund, at the discretion of the
organisers.

Examples of unacceptable harassment, intimidation, and bullying behaviour

Harassment includes, but is not limited to:

- Making comments, to an audience or personally, that belittle or demean another person
- Sexual images in public spaces
- Stalking or following
- Harassing photography or recording
- Sustained disruption of talks or other events
- Inappropriate physical contact
- Unwelcome sexual attention
- Advocating for, or encouraging, any of the above behaviour

Intimidation and bullying include, but are not limited to:

- Aggressive or browbeating behaviour directed at someone during a public presentation
- Mocking or insulting another person’s intellect, work, perspective, or question/comment
- Making reference to someone’s gender, gender identity and
  expression, sexual orientation, disability, physical appearance,
  body size, race, age, religion, or other personal attribute in the
  context of a scientific discussion
- Deliberately making someone feel unwelcome

#### Enforcement

Participants asked to stop any harassing behaviour are expected to
comply immediately.

If a participant engages in harassing behaviour, event organisers
retain the right to take any actions to keep the event a welcoming
environment for all participants. This includes warning the offender
or expulsion from the event with no refund.

Event organisers may take action to redress anything designed to, or
with the clear impact of, disrupting the event or making the
environment hostile for any participants.

We expect participants to follow these rules at all event venues and
event-related social activities. We think people should follow these
rules outside event activities too!  Reporting

If someone makes you or anyone else feel unsafe or unwelcome, please
report it as soon as possible. Harassment and other code of conduct
violations reduce the value of our event for everyone. We want you to
be happy at our event. People like you make our event a better place.

#### Personal Report

You can make a personal report with Laurent Gatto or Axelle
Loriot. You can contact them directly or by email at
`name.surname@uclouvain.be`.

When taking a personal report, our staff will ensure you are safe and
cannot be overheard. They may involve other event staff to ensure your
report is managed properly. Once safe, we’ll ask you to tell us about
what happened. This can be upsetting, but we’ll handle it as
respectfully as possible, and you can bring someone to support
you. You won’t be asked to confront anyone and we won’t tell anyone
who you are.

We value your attendance.

## Acknowledgement

This summer school is organised in collaboration with GSK.

We would like to acknowledge financial support from GSK and thank the
UCLouvain for administrative and logistic support.

<img height="70" src="figs/UCLouvain.jpg">
<img height="70" src="figs/GSK.jpg">
